ros.ws/src/docker/docker-compose.yml

```
@@ -0,0 +1,52 @@
+      - ./gauge_net/gauge_net/models:/ros.ws/src/gauge_net/gauge_net/models
+    environment:
+      - ROS_DOMAIN_ID=10
+      - RMW_IMPLEMENTATION=rmw_cyclonedds_cpp
+    network_mode: "host"  # Required for ROS2 discovery and robot communication
+    privileged: true  # Required for hardware access
+    restart: unless-stopped
+    command:  ["ros2", "launch", "gauge_net", "gauge_net.launch.py"]
+  webrtc:
+    image: go2_ros_sdk:latest
+    environment:
+      - ROBOT_IP=${ROBOT_IP} 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)